genrl/deep/common/trainer.py

```diff
@@ -381,22 +381,7 @@ def train(self) -> None:
                     and timestep % self.update_interval == 0
                 ):
                     for _ in range(self.update_interval):
```

Comment:
Since we're using that `_`, I think it should be some letter.

This seems to be an issue that was created previously though

Reply:
yea this i'll do